src/utils/sortMovie.ts

```diff
@@ -0,0 +1,61 @@
+import { MockData } from '../mockDate/date';
+
+enum SORT_RULE {
+  SORT_A_BEFORE_B = -1,
+  KEEP_ORIGINAL,
+  SORT_B_BEFORE_A,
+}
+
```

Comment:
what is it?

Reply:
If I remember the task correctly, this is an extra file.

Reply:
removed, thanks